src/MainWindow.cpp

```diff
@@ -1740,3 +1760,28 @@ void MainWindow::on_pushButtonSubDomain_clicked()
         checkSubdomainSelection();
     }
 }
+
+void MainWindow::on_pushButtonHIBP_clicked()
+{
+    QSettings s;
+
+    bool en = s.value("settings/enable_hibp_check", false).toBool();
+
+    int ret;
+    if (en)
+    {
+        ret = QMessageBox::question(this, "Moolticute", tr("Disable HIBP check?"));
```

Comment:
The name HIBP is not common, user will get confused, please use the full service name